doc/make/index.md

```diff
@@ -88,6 +88,18 @@ Pass these along with the `cmake ../../src` command.
   every `git commit`. Use this option to avoid the version check. The `.olean`
   files can be removed manually by invoking `make/ninja clean-olean`.
 
+Further Tips For Developers
+---------------------------
+
+In the below tips you can replace `make` with `ninja` as needed.
+
+* To save some time when compiling: use `make shell/lean` to only compile the things needed to run lean (no tests are built).
```

Comment:
Do you mean `make bin_lean` here? (`make shell/lean` doesn't work for me on macOS)

Reply:
Hmm... maybe the `standard_lib` target should be mentioned here instead actually, since we haven't turned off the olean version checking (yet?).

Reply:
I think it's either `make bin_lean` or `ninja shell/bin_lean`.